components/huds/Hud.js

```diff
@@ -35,7 +35,7 @@ function switchColor(player, winner) {
   }
 }
 
-function Hud({ player, winner }) {
+function Hud({ player, winner, props }) {
```

Comment:
No, you're declaring a prop named `props`. Use the rest operator to get all undeclared props in an array named `props`:

```suggestion
function Hud({ player, winner, ...props }) {
```

Reply:
Fixed and understood :)